flycheck.el

```diff
@@ -2645,7 +2655,7 @@ to enable disabled checkers.")))
       (fill-region-as-paragraph (point) end)))
 
   (princ "\n\n--------------------\n\n")
-  (princ (format "Flycheck version: %s\n" (flycheck-version)))
+  (princ (format "Flycheck version: %s\n" (flycheck--pkg-version)))
```

Comment:
I guess you don't really need to change this, as `flycheck-version` still will return this as a string.